Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN opam install -y --deps-only .
 ADD --chown=opam . .
 RUN opam config exec -- dune build ./_build/install/default/bin/ocurrent-deployer
 
-FROM debian:10
+FROM debian:11
 RUN apt-get update && apt-get install libffi-dev libev4 openssh-client curl gnupg2 dumb-init git graphviz libsqlite3-dev ca-certificates netbase rsync -y --no-install-recommends
 RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
 RUN echo 'deb [arch=amd64] https://download.docker.com/linux/debian buster stable' >> /etc/apt/sources.list
```

Comment:
Just thought of this while re-reading the PR, maybe I should update `buster` to `bullseye` since I updated the Dockerfile from Debian 10 to 11?

Reply:
Yes, that needs to match.